models/demos/tg/llama3_70b/tt/model_config.py

```diff
@@ -347,16 +368,16 @@ def set_mlp_config(model_config, cluster_shape):
         orientation=ttnn.ShardOrientation.ROW_MAJOR,
         use_height_and_width_as_shard_shape=True,
     )
-    decode_config["FF1_OUT_GATHERED_MEMCFG"] = ttnn.create_sharded_memory_config(
-        shape=(M * cluster_shape[0], N // 8),
-        core_grid=ttnn.CoreGrid(y=1, x=8),
+    decode_config["FF1_OUT_REDUCE_SCATTER_MEMCFG"] = ttnn.create_sharded_memory_config(
+        shape=(M, N // 28 // cluster_shape[0]),
```

Comment:
Ideally use variable num cores (instead of 28 hard coded) / and generate core range from that number, or at least comment what the 28 is.

Reply:
Yeah, agree on the point, actually the entire model_config file need parameterizations. Have been just following the pattern as we are gonna move to new codebase

Reply:
Will add a comment